apps/src/aichat/views/presentation/PresentationView.tsx

```diff
@@ -1,40 +1,67 @@
-import React from 'react';
+import React, {useMemo} from 'react';
 import ModelCardRow from './ModelCardRow';
-import {MODEL_CARD_FIELDS_AND_LABELS} from '@cdo/apps/aichat/views/modelCustomization/constants';
+import {
+  MODEL_CARD_FIELDS_LABELS_ICONS,
+  TECHNICAL_INFO_FIELDS,
+} from '@cdo/apps/aichat/views/modelCustomization/constants';
 import styles from '@cdo/apps/aichat/views/model-customization-workspace.module.scss';
 import {Heading4} from '@cdo/apps/componentLibrary/typography';
 import moduleStyles from './presentation-view.module.scss';
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
 
 const PresentationView: React.FunctionComponent = () => {
+  const currentAiCustomizations = useAppSelector(
+    state => state.aichat.currentAiCustomizations
+  );
+  const {botName, systemPrompt, temperature, retrievalContexts} =
+    currentAiCustomizations;
+  const modelCardInfo = currentAiCustomizations.modelCardInfo;
+
+  // These are temporary constants. They will be retrieved from s3.
+  const EXAMPLE_MODEL_NAME = 'Model A';
+  const EXAMPLE_MODEL_TRAINING_DATA = 'Model A Training Data';
+
+  const technicalInfo = useMemo(() => {
+    const TECHNICAL_INFO: {
```

Comment:
nit: lowercase since this is a local var? (maybe `technicalInfoData` or something)